__tests__/app.test.js

```diff
@@ -56,6 +56,7 @@ describe("/api/articles/:article_id", () => {
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: expect.any(Number),
```

Comment:
It would have been more in line with _Good Practice TDD_ to add this as an additional test, rather than going back to modify a previous test 👍